__tests__/app.test.js

```diff
@@ -122,4 +130,58 @@ describe('GET /api/articles', () => {
 				expect(articles).toBeSortedBy('created_at', { descending: true });
 			})
 	})
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles/:article_id/comments', () => {
+
+	test('GET:200 should return an array of comments for the given article_id', () => {
+		return request(app)
+			.get('/api/articles/1/comments')
+			.expect(200)
+			.then((response) => {
+				const comments = response.body.comments;
+				expect(comments).toEqual(expect.any(Array));
+
+				expect(comments).toEqual(
+					expect.arrayContaining([
+						expect.objectContaining({
+							comment_id: expect.any(Number),
+							body: expect.any(String),
+							author: expect.any(String),
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+							article_id: expect.any(Number),
+						})
+					])
+				)
+			})
+	});
+	test('comments should be served with the most recent comments first', () => {
+		return request(app)
+			.get('/api/articles/1/comments')
+			.expect(200)
+			.then((response) => {
+				const comments = response.body.comments;
+				console.log(comments);
+				expect(comments).toBeSortedBy('created_at', { descending: true });
+			});
+	});
+
+	test('GET:400 should handle invalid article_id format', () => {
+		return request(app)
+			.get('/api/articles/not-an-article/comments')
+			.expect(400)
+			.then((response) => {
+				expect(response.body.message).toBe('Invalid article_id');
+			});
+	});
+
+	test('GET:404 should handle article not found', () => {
+		return request(app)
+			.get('/api/articles/999/comments')
+			.expect(404)
+			.then((response) => {
+				expect(response.body.message).toBe('Article not found');
+			});
+	});
```

Comment:
Please add a test for when the article ID is valid but it has no comments and so responds with an empty array

Reply:
done